include/seqan3/utility/views/chunk.hpp

```diff
@@ -574,7 +574,10 @@ class chunk_view<urng_t>::basic_iterator : public maybe_iterator_category<maybe_
     constexpr basic_iterator & operator+=(difference_type const skip) noexcept
         requires std::random_access_iterator<it_t>
     {
-        auto new_start_it = current_chunk.begin() + (chunk_size * skip);
+        if (skip < 0)
+            return operator-=(-skip);
+
+        auto new_start_it = std::ranges::next(current_chunk.begin(), chunk_size * skip, urng_end);
```

Comment:
I think this is fine. But it is weird that you didn't need to adjust `operator-=`. I expect the same error if skip is negative. Can you add the same change to `operator-=` (delegating to `operator+=` if skip is negative) and see if all tests run through? maybe our tests just don't catch this.